portfolio/src/main/webapp/index.html

```diff
@@ -31,13 +31,22 @@ <h1>Lily Zhou</h1>
         <button class="button"><a href="images.html">Food Journal</a></button>
         <button class="button"><a href="projects.html">Projects</a></button>
       </div>
-      <form action="/data" method="POST">
-        <textarea name="name-input">Name</textarea>
-        <textarea name="comment-input">Your comment here.</textarea>
-        <br/><br/>
-        <input type="submit" />
-      </form>
-      <div id="comments-container"></div>
+      <div class="comments-box">
```

Comment:
this is not part of this PR, correct?

Reply:
oops yes -- I think this was part of a commit from last week's project, and was reviewed in a previous pull request.